region/crateria/central.json

```diff
@@ -3394,6 +3394,56 @@
                   ],
                   "note": "Does not kill the boyons but does remove them as an obstacle."
                 },
+                {
+                  "name": "Frozen Boyon Runway And Fast Walljump",
+                  "notable": false,
+                  "requires": [
+                    "canUseFrozenEnemies",
+                    "canShinechargeMovementComplex",
+                    "canFastWalljumpClimb",
+                    {"canShineCharge": {
+                      "usedTiles": 33,
+                      "shinesparkFrames": 98,
+                      "excessShinesparkFrames": 6,
+                      "openEnd": 2
+                    }}
+                  ],
+                   "obstacles": [
+                    {
+                      "id": "A",
+                      "requires": ["Ice"]
```

Comment:
Thinking about this more, this won't work. If you kill the Boyons with Supers for the other strat, then ice is no longer needed in this strat. To fix it, ice should be in the main part and the obstacle section can be removed (or in there with the requires empty).

Reply:
Ice is required in the `canUseFrozenEnemies`, but writing out obstacles everytime when its a free pass looks pretty ugly imo

Reply:
Oh, right. I would be fine either way on this. I would probably not include the obstacle